pkg/diskmaker/controllers/lvset/reconcile.go

```diff
@@ -82,10 +90,41 @@ func (r *ReconcileLocalVolumeSet) Reconcile(request reconcile.Request) (reconcil
 	provisionerConfig := staticProvisioner.ProvisionerConfiguration{}
 	staticProvisioner.ConfigMapDataToVolumeConfig(cm.Data, &provisionerConfig)
 
+	r.runtimeConfig.DiscoveryMap = provisionerConfig.StorageClassConfig
+	r.runtimeConfig.NodeLabelsForPV = provisionerConfig.NodeLabelsForPV
+	r.runtimeConfig.UseAlphaAPI = provisionerConfig.UseAlphaAPI
+	r.runtimeConfig.UseJobForCleaning = provisionerConfig.UseJobForCleaning
+	r.runtimeConfig.MinResyncPeriod = provisionerConfig.MinResyncPeriod
+	r.runtimeConfig.UseNodeNameOnly = provisionerConfig.UseNodeNameOnly
+	r.runtimeConfig.Namespace = request.Namespace
+	r.runtimeConfig.LabelsForPV = provisionerConfig.LabelsForPV
+	r.runtimeConfig.SetPVOwnerRef = provisionerConfig.SetPVOwnerRef
+	r.runtimeConfig.Name = getProvisionedByValue(*r.runtimeConfig.Node)
+
```

Comment:
I think this is a big surface area to copy from static-provisioner and prone to cause problems if they change their interfaces or we will be stuck with version we currently use.

Reply:
It's mostly required configuration for the deleter.  It's all a struct representation of their configuration file format, so we should be guaranteed some stability. That being said, we can probably get rid of these:
```
	r.runtimeConfig.UseAlphaAPI = provisionerConfig.UseAlphaAPI
	r.runtimeConfig.NodeLabelsForPV = provisionerConfig.NodeLabelsForPV
	r.runtimeConfig.UseJobForCleaning = provisionerConfig.UseJobForCleaning
	r.runtimeConfig.MinResyncPeriod = provisionerConfig.MinResyncPeriod
```
and hardcode these:
```
	r.runtimeConfig.UseNodeNameOnly = false
	r.runtimeConfig.SetPVOwnerRef = true
```

Reply:
First time we're bitten by this, we can switch to not using runtimeconfig inside our own methods.